geonode/static/lib/js/autocomplete.js

```diff
@@ -0,0 +1,100 @@
+
```

Comment:
As far as I can see you introduced a new static JS file which is not part of the "package.json" nor of the "static_dependencies.json".
That means that trying to build GeoNode assets from scratch this will be completely lost.

Question:
 - is this a custom JS or it is part of a common library?

If this is a custom JS this should be placed under "static/geonode/js/..." instead.

Reply:
Yes, this is custom JS. It is used for the text input autocompletes used by all the searches (not the multiselects which is covered by select2). Thinking it can can be moved to the "static/geonode/js/search" directory rather than making a separate '/autocomplete directory.

Reply:
Yeps, go ahead